plugins.yml

```
@@ -105,6 +105,8 @@ plugins:
             'guide/page/create_custom_page_block.md': 'content_management/pages/create_custom_page_block.md'
             'guide/form_builder/forms.md': 'content_management/forms/forms.md'
             'api/public_php_api_managing_forms.md': 'content_management/forms/form_api.md'
+            'guide/form_builder/work_with_forms.md': 'content_management/forms/work_with_forms.md'
+            'api/public_php_api_managing_work_with_forms.md': 'content_management/forms/form_api.md'
```

work_with_forms.md is a new file so it doesn't have previous locations.

There are two renamings to backup as redirections

- docs/ibexa_products/experience.md → docs/ibexa_products/ibexa_experience.md
- docs/ibexa_products/headless.md → docs/ibexa_products/ibexa_headless.md

```suggestion
            'docs/ibexa_products/experience.md': 'docs/ibexa_products/ibexa_experience.md'
            'docs/ibexa_products/headless.md': 'docs/ibexa_products/ibexa_headless.md'
```